sites/public/lib/hooks.ts

```diff
@@ -109,3 +114,23 @@ export async function fetchBaseListingData() {
 
   return listingData
 }
+
+let jurisdiction: Jurisdiction
+
+export async function fetchJurisdictionByName() {
+  try {
+    if (jurisdiction) {
+      return jurisdiction
+    }
+
+    const jurisdictionName = process.env.jurisdictionName
+    const jurisdictionRes = await axios.get(
```

Comment:
Why not use an autogenerated client service here?

Reply:
Because it actually feels like more work to do that, since we have to set an axios instance to use it anyway. We have this handled in AuthContext, but that's used for React hooks, so we need to setup a different more reusable function to make it easier. I'll create task to do this. Since we have shared-helpers, that would be a good place to add it.